test_scripts/Polices/build_options/032_ATF_P_TC_PTU_Retry_Sequence_Retry_Timeout_Expiration_HTTP.lua

```diff
@@ -75,42 +71,45 @@ function Test:TestStep_OnStatusUpdate_UPDATE_NEEDED_new_PTU_request()
   local correlationId = self.mobileSession:SendRPC("RegisterAppInterface", config.application1.registerAppInterfaceParams)
 
   EXPECT_HMINOTIFICATION("BasicCommunication.OnAppRegistered", {application = { appName = config.application1.registerAppInterfaceParams.appName } })
-  EXPECT_HMINOTIFICATION("SDL.OnStatusUpdate",
-    { status = "UPDATE_NEEDED" }, {status = "UPDATING"}):Times(2)
-  EXPECT_NOTIFICATION("OnSystemRequest", {requestType = "HTTP"})
-  :Do(function()
-    time_system_request_prev = timestamp()
-  end)
+  EXPECT_HMINOTIFICATION("SDL.OnStatusUpdate",{ status = "UPDATE_NEEDED" }, {status = "UPDATING"}):Times(2)
+
+  EXPECT_NOTIFICATION("OnSystemRequest")
+  :Do(function(_, data)
+      print("SDL->MOB: OnSystemRequest()", data.payload.requestType)
+      if data.payload.requestType == "HTTP" then
+        time_system_request_prev = timestamp()
+      end
+    end)
+  :Times(2)
 
   self.mobileSession:ExpectResponse(correlationId, { success = true, resultCode = "SUCCESS" })
   self.mobileSession:ExpectNotification("OnHMIStatus", {hmiLevel = "NONE", audioStreamingState = "NOT_AUDIBLE", systemContext = "MAIN"})
 end
 
---[[ Test ]]
-commonFunctions:newTestCasesGroup("Test")
-
 function Test:TestStep_Retry_Timeout_Expiration()
   local is_test_fail = false
   local timeout_after_x_seconds = 30
   local time_wait = {}
   local sec_btw_ret = {1, 2, 3, 4, 5}
   local total_time
 
-  time_wait[1] = timeout_after_x_seconds
-  time_wait[2] = sec_btw_ret[1] + timeout_after_x_seconds
-  time_wait[3] = sec_btw_ret[1] + sec_btw_ret[2] + timeout_after_x_seconds
-  time_wait[4] = sec_btw_ret[2] + sec_btw_ret[3] + timeout_after_x_seconds
-  time_wait[5] = sec_btw_ret[3] + sec_btw_ret[4] + timeout_after_x_seconds
-  time_wait[6] = sec_btw_ret[4] + sec_btw_ret[5] + timeout_after_x_seconds
+  time_wait[1] = timeout_after_x_seconds -- 30
+  time_wait[2] = timeout_after_x_seconds + sec_btw_ret[1] -- 31
```

Comment:
@istoimenova, According to "Formula for retry sequence timeout" clarification implemented formula is correct.